.changelog/16647.txt

```diff
@@ -0,0 +1,4 @@
+integration test to cover the use case of taking and restoring a snapshot on a consul cluster
```

Comment:
Do we normally add testing items to the changelog? I'm not even quite sure what happens to text outside of the release note.

Reply:
yea, that was a remnant that was not meant to be there.  I removed it.